lib/smart_proxy_remote_execution_ssh/runners/script_runner.rb

```diff
@@ -195,27 +195,28 @@ def prepare_start
       @output_path = File.join(File.dirname(@remote_script), 'output')
       @exit_code_path = File.join(File.dirname(@remote_script), 'exit_code')
       @pid_path = File.join(File.dirname(@remote_script), 'pid')
+      su_method = @user_method.instance_of?(SuUserMethod)
+      wrapper = <<~SCRIPT
+        if [ "$1" == "inner" ]; then
+          echo \$$ > #{@pid_path}
+          (
+            #{@user_method.cli_command_prefix}#{su_method ? "'exec #{@remote_script} < /dev/null '" : "#{@remote_script} < /dev/null"}
+            echo \$? >#{@exit_code_path}
+          ) | tee #{@output_path}
+        else
+          exec #{UNSHARE_PREFIX} "$0" inner
```

Comment:
this will need something like
```suggestion
          UNSHARE=''
          if #{UNSHARE_PREFIX} true >/dev/null 2>/dev/null; then
              UNSHARE='#{UNSHARE_PREFIX}'
          fi
          exec $UNSHARE "$0" inner
```